build.gradle

```diff
@@ -1,27 +1,5 @@
-import de.undercouch.gradle.tasks.download.Download
```

Comment:
This import line needs to be kept in build_forge.gradle to avoid this:
<img width="386" alt="image" src="https://github.com/MinecraftForge/MinecraftForge/assets/3158390/cc64d1cd-b89d-4e20-9b57-2e2114370233">